### DISCUSSION TITLE
How to Modify Timestamps of Metrics in Signoz Stored in ClickHouse?

### DISCUSSION THREAD

**Discussion by FarzanaJade** [SigNoz/signoz#6509] [Q&A]
Hello Community,
I have a question: I want to modify the timestamp of metrics that I have already sent to Signoz and are stored in ClickHouse. If this is possible, could you please guide me on how to do it?

**Comment by srikanthccv** [comment]
ClickHouse doesn't do a good job with updates, so it's not a good idea.
